src/utils/innerSliderUtils.js

```diff
@@ -126,10 +126,7 @@ export const initializedState = spec => {
     currentSlide = slideCount - 1 - spec.initialSlide;
   }
   let lazyLoadedList = spec.lazyLoadedList || [];
-  let slidesToLoad = getOnDemandLazySlides(
-    { currentSlide, lazyLoadedList },
-    spec
-  );
+  let slidesToLoad = getOnDemandLazySlides({ currentSlide, lazyLoadedList });
```

Comment:
This is the real change I made. All other changes are automatic from the pre-commit hook.

Reply:
@bolatovumar  This could be the correct fix
```let slidesToLoad = getOnDemandLazySlides({ ...spec, currentSlide, lazyLoadedList });```

Can you check and make a change in this PR

Reply:
@akiran done